### ISSUE TITLE
Few problems with rowGrid.js

### ISSUE THREAD

**Issue by inliquid** [brunjo/rowGrid.js#19]
Hi!

Sorry, if this is not very right place for such report. In fact, rowGrid is embeded in CMS, which I use as the base for my website, but maintainers of CMS said that they can't help me as the problems I facing are likely to be on rowGrid side.

1) First of all rowGrid works great in 99.999% cases when the size of screen is large, lets say 1920px, in most browsers when there is no additional styles applied.

For example: http://artlab.club/graphic-kitchen-akaz/privetstvie-2.html

However I have an option when users apply their own branding to their blogs, meaning there is additional CSS in the page code. One of the significant changes (to my mind) is that there is 
```padding: 20px;```
style added to the div where photoset resides.

Examples of wrong alingment: 
http://artlab.club/radha/oranzhevoe-nastroenie.html

![1](https://cloud.githubusercontent.com/assets/6797881/8809447/6325dada-2ff1-11e5-930a-ab675860f109.png)

http://artlab.club/daria_miazhevich/pro-otnoshenie-k-printshopam.html

![1](https://cloud.githubusercontent.com/assets/6797881/9229721/27068ae4-4127-11e5-97f9-c3fda8e84b5d.png)

Alingment also depends on the screen scale and size, f.i. the picture above I got on all (17", 24") 1920x1080 monitors both in Chrome and Mozilla but with 100% scale. If I set screen zoom to 125% which is default in many Windows systems, it looks a little bit different (but still having this issue with alignment).
 
2) Secondly I'm still getting this issue, with tiny images but only in Chrome. I have changed this code:
```
function setRowGrid() {
           $('.js-topic-photoset-list').rowGrid({
               itemSelector: '.topic-photoset-item',
               minMargin: 10,
                maxMargin: 15,
               resize: false,
                lastRowClass: 'topic-photoset-last_row',
               firstItemClass: "first-item"
             });
         }
```
To this one:
```
function setRowGrid() {
           $('.js-topic-photoset-list img').load(function () {
               $('.js-topic-photoset-list').rowGrid({
                    itemSelector: '.topic-photoset-item',
                    minMargin: 10,
                    maxMargin: 15,
                    resize: false,
                    lastRowClass: 'topic-photoset-last_row',
                    firstItemClass: "first-item"
               });
             });
         }
```
And still getting this issue

http://artlab.club/letta-art/pervyj-mesyacz-risovaniya-v-365-itogi-i-razmyshleniya.html

![1](https://cloud.githubusercontent.com/assets/6797881/10223846/d5ee5b00-6863-11e5-96e9-9b88ddaa30a0.png)

http://artlab.club/radha/oranzhevoe-nastroenie.html

![2](https://cloud.githubusercontent.com/assets/6797881/10223923/2d1a18e2-6864-11e5-8b69-e98c44a6ddc0.png)

You need to press referesh several times to reproduce. 

However it seems that this problem is now only in Chrome. In FF I wasn't able to get to same results.

You may find that rowGrid.js is not the latest release, but I already tried latest 1.0.3 version and also tried to play with different options like min/max margin, resize, etc. I have test server and I could give you the link to see how it works there. (It's almost the same)

Please help, have now idea what to try else. 
Any advice would be appreciated......

**Comment by brunjo** [comment]
Hi,

you should add this to your CSS:
```CSS
.topic-photoset-item a {
  display: block;
}
```
I think this should fix the errors. If you still have problems don't hesitate to write again.

You should also set the `resize` option to `true` instead of `false` because you have a responsive design.

**Comment by inliquid** [comment]
Hi Bruno!

Thank you very much for your reply!

I have tried to apply this style and now I'm getting these pictures:

(All on 17" 1920x1080 Mozilla FireFox with 125% zoom - it's system wide scale in Windows, so FF sets this by default when viewing web pages - many people with FullHD screens have this setting. In Chrome it's still 100% by default and it looks better).

http://194.177.22.110/testtest/post-s-kartinkami.html

![1](https://cloud.githubusercontent.com/assets/6797881/10343169/3b440b92-6d25-11e5-9ae4-faf73de2f07e.png)

http://194.177.22.110/radha/oranzhevoe-nastroenie.html

![2](https://cloud.githubusercontent.com/assets/6797881/10343170/3b60780e-6d25-11e5-9859-e978401c9ba3.png)

This is my test machine - I have also updated rowGrid there to latest version and set 'resize' to 'true', as you recommended.
